protocols/gossipsub/src/protocol.rs

```diff
@@ -285,17 +287,21 @@ impl Decoder for GossipsubCodec {
     type Error = GossipsubHandlerError;
 
     fn decode(&mut self, src: &mut BytesMut) -> Result<Option<Self::Item>, Self::Error> {
+        // TODO: Replace with codec::decode call
```

Comment:
Once you have `prost_codec::Codec` as a field in `GossipsubCodec`, you can access its `decode` method here.